### PR TITLE
Update licensing across various source files - 1.14

### PR DESCRIPTION
**Pull request by stevendpclark**
Update various files from MPL-2.0 to BUSL-1.1 along with changing Linux packaging license to BUSL-1.1

### CODE REVIEW

File: command/server/config_test_helpers.go [hunk 1]
```diff
@@ -503,7 +503,7 @@ func testUnknownFieldValidation(t *testing.T) {
 			Problem: "unknown or unsupported field bad_value found in configuration",
 			Position: token.Pos{
 				Filename: "./test-fixtures/config.hcl",
-				Offset:   651,
+				Offset:   652,
```
Comment (mpalmi) [inline command/server/config_test_helpers.go:506]: Same as in the other review, is this change intentional?

Comment (stevendpclark) [reply command/server/config_test_helpers.go:506]: Yup it was intentional, the test code is leveraging the file position markers within command/server/test-fixtures/config.hcl so it needed to be updated. The same fix was done on 1.15/main

**Review by mpalmi — APPROVED** [review]
